addons/website/static/src/scss/website.scss

```diff
@@ -201,6 +201,33 @@ body {
         @include body-image-bg-style();
     }
 
+    > main {
+        > .page_breadcrumb {
+            a {
+                color: inherit !important;
+            }
+            > section {
+                @include o-apply-colors('breadcrumb');
+                @include o-apply-colors('breadcrumb-custom');
+                @include o-add-gradient('breadcrumb-gradient');
+            }
+        }
+    }
+
+    > main.o_breadcrumb_overlay {
+        > .page_breadcrumb {
+            @include o-position-absolute(auto, 0, auto, 0);
+            z-index: 99 !important;
```

Comment:
Why not 98? :wink: Use a value that make sense based on bootstrap z-index values. Why not the same as the header one?

Also, why not testing having a header over the content + a breadcrumb over the content? They are placed over each other... at minimum we should prevent being able to set them both as "over the content" or, better(?), support everything? cc @lebl-odoo ?
![image](https://github.com/user-attachments/assets/990ec289-9bea-454f-aa6f-19dbb2fc5607)

Reply:
Indeed, we should compute the relative position of the breadcrumb regarding the position of the header and the settings of those two elements. We should support all possible combination.

If I use the standard behavior for header and set "over the content" for breadcrumb, it seems it doesn't work as expected.

Reply:
@qsm-odoo We need to compute height of the header and need to set margin-top for Bredcrumb element, in case of over the content header style, header position will be absolute, so it will header will be out of the document flow.

cc: @lebl-odoo

Reply:
@msh-odoo There are already code that does this kind of things: for example, the table of content menu when it is on top of the content. You should implement this the same way :pray: